index.mjs

```diff
@@ -33,61 +35,63 @@ import LookupUSEnrichment from "./src/us_enrichment/Lookup.js";
 import ResponseUSEnrichment from "./src/us_enrichment/Response.js";
 
 export const core = {
-  Batch,
-  ClientBuilder,
-  buildClient,
-  SharedCredentials,
-  StaticCredentials,
-  Errors,
+	AgentSender,
```

Comment:
I don't think that the `AgentSender` and `Request` classes should be exposed as part of the `core`. Our end users shouldn't be calling those directly so they shouldn't be included in `core`.